core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -439,6 +440,14 @@ class CudaInternal {
     return cudaGraphInstantiate(pGraphExec, graph, pErrorNode, pLogBuffer,
                                 bufferSize);
   }
+#else
+  template <bool setCudaDevice = true>
+  cudaError_t cuda_graph_instantiate_wrapper(cudaGraphExec_t* pGraphExec,
+                                             cudaGraph_t graph) const {
+    if constexpr (setCudaDevice) set_cuda_device();
+    return cudaGraphInstantiate(pGraphExec, graph);
```

Comment:
Provide only the interface without the extra trailing arguments.
```suggestion
#if CUDA_VERSION < 12000
    constexpr size_t error_log_size = 256;
    cudaGraphNode_t error_node      = nullptr;
    char error_log[error_log_size];
    return cudaGraphInstantiate(pGraphExec, graph, &error_node, error_log,
                                error_log_size);
#else
    return cudaGraphInstantiate(pGraphExec, graph);
#endif
```

Reply:
Done when doing the suggestion above.